iscservice/ut/service_ut.c

```diff
@@ -142,11 +141,10 @@ static void fid_get(const char *f_name, struct m0_fid *fid)
 
 int isc_ut_server_start(void)
 {
-	int rc = 0;
-
+	int                 rc = 0;
```

Comment:
No need to change the variable alignment.

Reply:
incorporated